app/components/user-menu.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  classNames: ['user-menu'],
+  classNames: ['user-menu', 'dropdown'],
```

Comment:
I like to keep a component-specific class name around just for the sake of easier test writing. Not sure where you stand on that.